src/index.html

```diff
@@ -8,6 +8,58 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="box">
+      <div class="star-block stars--0">
```

Comment:
```suggestion
      <div class="stars stars--0">
```
must be the block stars